controlplane/hack/deploy/addon/policy/propagator/grc-policy-propagator-rolebinding.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  labels:
+    app: grc
+    app.kubernetes.io/instance: grc
+    app.kubernetes.io/name: grc
+    chart: grc-chart-2.7.0
+    component: ocm-grc-rolebinding
+    release: grc
+  name: grc-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: grc-role
+subjects:
+- kind: ServiceAccount
+  name: grc-sa
```

Comment:
if the name of sa is not default,  kustomize edit namespace cannot add namespace the sa in rolebinding.

Reply:
Iin my env, the sa namespace was added
```
apiVersion: rbac.authorization.k8s.io/v1
kind: RoleBinding
metadata:
  annotations:
    kubectl.kubernetes.io/last-applied-configuration: |
      {"apiVersion":"rbac.authorization.k8s.io/v1","kind":"RoleBinding","metadata":{"annotations":{},"labels":{"app":"grc","app.kubernetes.io/instance":"grc","app.kubernetes.io/name":"grc","chart":"grc-chart-2.7.0","component":"ocm-grc-rolebinding","release":"grc"},"name":"grc-rolebinding","namespace":"hub1"},"roleRef":{"apiGroup":"rbac.authorization.k8s.io","kind":"Role","name":"grc-role"},"subjects":[{"kind":"ServiceAccount","name":"grc-sa","namespace":"hub1"}]}
  creationTimestamp: "2022-11-17T10:17:24Z"
  labels:
    app: grc
    app.kubernetes.io/instance: grc
    app.kubernetes.io/name: grc
    chart: grc-chart-2.7.0
    component: ocm-grc-rolebinding
    release: grc
  name: grc-rolebinding
  namespace: hub1
  resourceVersion: "33433"
  uid: ced7df68-8865-47d8-a146-63ecbfc03d45
roleRef:
  apiGroup: rbac.authorization.k8s.io
  kind: Role
  name: grc-role
subjects:
- kind: ServiceAccount
  name: grc-sa
  namespace: hub1
```

Reply:
I think it works because I didn't specify name of referenced serviceaccount.

Reply:
yes, it works.